libs/cdk/package.json

```
@@ -48,7 +48,7 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": ">=12.0.0",
+    "@angular/core": ">=13.0.0",
```

issue: Only mark Angular 13.x as officially supported
```suggestion
    "@angular/core": "^13.0.0",
```

---

It's not really telling the truth. You can easily use this version with any version above 13. But on the other hand, please do what feels right. I just want to get this thing in!! :D

---

as far as angular repo itself uses this format we can go in the same way

https://github.com/angular/angular/blob/main/packages/router/package.json#L30

`json
"peerDependencies": {
  "rxjs": "^6.5.3 || ^7.4.0"
}
`